firebase.js

```diff
@@ -284,35 +294,45 @@ const config = {
         type: 302,
       },
       {
-        destination: '/geoip/legacy/codes',
+        destination: '/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired',
         source: '/geoip/legacy/codes/country_continent{,/}',
```

Comment:
We can probably condense this so that anything starting with `/geoip/legacy/**` gets redirected